src/AspNetCore/nativeaot.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Does this script work on Windows?

Reply:
This exact script obviously not. You have to change `linux-x64` to `win-x64` and that's it, but this specific project has not `run.cmd` either, so I decide follow what's there. It's not a problem to add similar scripts.

Reply:
If I remember correctly I _can_ build/publish .NET app with any runtime identifier from Windows or any other OS. My question is more about `strip` command. Or do you mean that I can use aot compilation only when building project from the same platform?

Reply:
Cross compilation for NativeAOT does not supported.
`strip` command needed only in Linux since for Windows ILC produce PDB file which separate from executable.

Reply:
So for Windows script still _should_ remove pdbs but with other command, right? If so I propose to add another script and add some notes.

Reply:
On windows PDB is separate file from EXE. So you can just `del` them :smile: .  On linux symbols information is part of ELF file. So we have to seprately strip to show actual size (otherwise people would be confused about real size of the app). 
Let me add nativeaot.cmd to show you the steps.

Reply:
If you write script to automate process then no need to force people delete not needed (usually) files - that what I mean.

Reply:
I think there some misunderstanding what happens with `strip`. In the original commit, I just highlight possibility to strip debugging symbols from resulting file(file still present) to reduce size. Maybe docs will explain better then me https://github.com/dotnet/runtimelab/blob/feature/NativeAOT/docs/using-nativeaot/optimizing.md#special-considerations-for-linuxmacos

Reply:
That is, the presence of  PDBs does not oblige to distribute them, it already depends on which files to include in the app distribution. Then OK.

Reply:
1. There no PDB files on Linux when compiling using NativeAOT. There on Windows. 
2. Linux by default provide ELF files with embedded debugging symbols which increase file size. That's not obvious.
3. That's why I provide information about command which maybe of interest to new developer.

By default on Linux you would distribute larger files then necessary. I prefer to keep default publishing behavior in the example and only give a hint (in comments) that some additional steps may be nescessary.

I'm fine with `strip` or without it.